frameworks/PHP/phalcon/public/index-micro.php

```diff
@@ -96,8 +101,9 @@
         ));
 
     });
-    $request = new Phalcon\Http\Request();
-    $app->handle($request->getURI());
+    
+    $url = $_REQUEST['_url'] ?? '/';
```

Comment:
`Phalcon\Http\Request()` works fine, it's only this particular case calling the handler; it's not been used anywhere else on the test, but if it would, it shouldn't be a problem.

Reply:
Normally all frameworks in this bench use a realistic approach, and try to avoid an striped version.